beps/bep_0007.rst

```diff
@@ -23,32 +24,35 @@ be interpreted as described in IETF `RFC 2119`_.
 
 .. _`RFC 2119`: http://tools.ietf.org/html/rfc2119
 
-Announce Parameter
-==================
+Announcing
+==========
+
+For each local IP address the client:
+
+	i. intends to receive incoming peer connections over (listens on)
+	ii. intends to publish to the tracker
+	iii. can route packets to the tracker via
```

Comment:
I have difficulty parsing this sentence. I get its point, but the word order seems wrong.

Reply:
yeah, re-reading it now, it sounds a bit awkward. it ends up being: "For each local IP address the client can route packets to the tracker via".

With the first part fixed to "For each local IP address the client", it's not trivial to come up with a different ending. I suppose I could factor-in "the client" into the two first bullets, and say: "that can route packets to the tracker". Maybe that's better, I'll try that.

Reply:
I think "can be used as the source address in packets sent to the tracker" works